x-pack/legacy/plugins/siem/public/containers/matrix_histogram/index.ts

```diff
@@ -109,16 +94,7 @@ export const useQuery = <Hit, Aggs, TCache = object>({
       isSubscribed = false;
       abortCtrl.abort();
     };
-  }, [
-    defaultIndex,
-    query,
-    filterQuery,
-    isInspected,
-    isDnsHistogram,
-    stackByField,
-    startDate,
-    endDate,
-  ]);
+  }, [defaultIndex, filterQuery, isInspected, stackByField, startDate, endDate, data]);
```

Comment:
It appears that following should also be listed as dependencies:

- `errorMessage`
- `histogramType`